docs/api-opentelemetry.asciidoc

```diff
@@ -0,0 +1,121 @@
+ifdef::env-github[]
+NOTE: For the best reading experience,
+please view this documentation at https://www.elastic.co/guide/en/apm/agent/dotnet[elastic.co]
+endif::[]
+
+[[opentelemetry-bridge]]
+== OpenTelemetry Bridge
+
+NOTE: Added as experimental in v1.13.0.
+To enable it, set <<config-enable-open-telemetry-bridge, `OpentelemetryBridgeEnabled`>> to `true`.
+
+NOTE: The OpenTelemetry bridge is supported on .NET 5 and newer versions.
+
+The OpenTelemetry Bridge in the Elastic .NET APM Agent bridges OpenTelemetry spans into Elastic APM transactions and spans. The Elastic APM OpenTelemetry bridge allows one to use the vendor-neutral OpenTelemetry Tracing API to manually instrument one's code, and have the Elastic .NET APM agent handle those API calls. This allows one to use the Elastic APM agent for tracing, without any vendor lock-in from adding manual tracing using the APM agent’s own <<public-api, Public API>>.
+
+OpenTelemetry in .NET is implemented via the https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.activity?view=net-5.0[Activity API].
+
+In .NET, there is an https://opentelemetry.io/docs/instrumentation/net/shim/[OpenTelemetry shim] which is built on top of the Activity API. 
+
+The OpenTelemetry bridge in the Elastic .NET APM Agent targets the https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.activity?view=net-5.0[Activity API]. Since the https://opentelemetry.io/docs/instrumentation/net/shim/[OpenTelemetry .NET shim] builds on top of the https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.activity?view=net-5.0[Activity API], the shim is implicitly supported as well, although we don't directly test it, because the Activity API is the recommended API.
```

Comment:
A couple of thoughts here.

1. This seems repetitive
2. Are these implementation details?

Can we move this to the end? Maybe a "how this feature works" section?

Reply:
I moved it into the end.

What I wanted to say here is this:
- OTel has its predefined API (like `Tracer`, `Span`, etc.) - of course .NET does not follow this :D In .NET there is this thing called `Activity` which is the equivalent of a span, so the API (especially the name of specific types, and functions) for .NET OTel is different - things are naming differently and the .NET implementation doesn't follow the naming of the spec. (I know... not nice, but that's how it is). So I wanted to say that when we talk about the OTel bridge in .NET, what we mean is that we bridge the `Activity` API (which is de-facto OTel in .NET).
- To make things more complicated there is actually an alternative API that strictly follows the spec - that's the "shim" I talk about. That's built on top of `Activity`, and this shim is not recommended to use (the doc of it literally says "use `Activity` instead"). But I wanted to point out, that we indirectly support this shim as well.

> Are these implementation details?

So I would not say this is implementation detail - I'm just trying to communicate what API we actually support by the bridge.

I named the new section at the end `Supported OpenTelemetry implementations` - I think that better describes what this is.

> This seems repetitive

I changed the text a little bit.

Reply:
Aha! Thanks. This makes sense to me now. I like it at the end if you're okay with it there. And the rewording is great.